packages/paste-core/components/chat-composer/src/ChatComposerAttachment.tsx

```diff
@@ -0,0 +1,49 @@
+import type { BoxElementProps } from "@twilio-paste/box";
+import { Box } from "@twilio-paste/box";
+import { MediaBody, MediaFigure, MediaObject } from "@twilio-paste/media-object";
+import { Stack } from "@twilio-paste/stack";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+import * as React from "react";
+
+export interface ChatComposerAttachmentProps extends HTMLPasteProps<"div"> {
+  children: NonNullable<React.ReactNode>;
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   *
+   * @default "CHAT_COMPOSER_ATTACHMENT"
+   * @type {BoxProps["element"]}
+   * @memberof ChatComposerAttachmentProps
+   */
+  element?: BoxElementProps["element"];
+  /**
+   * Pass an icon to use for the attachment message. DownloadIcon recommended
```

Comment:
question(totally non-blocking): not really about this PR, but why do we recommend and use the DownloadIcon? wouldn't the FileIcon, or specific file type icons be more relevant? is this something we'd want to think about at some point or nah?

Reply:
This was one of the components I copied over from the chat log component. @serifluous or @roxanagomez any ideas on this?